Experiment automation/(LUMICKS) Reference kit automation/referencekit_automation.py

```diff
@@ -0,0 +1,335 @@
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+--------------------------------
+This script Iteratively catches beads, fishes for DNA and makes a kymograph at 10 pN.
+The script is optimized for DNA tethers with a length of 37.8 kbp (12.85 micron).
+Before running the script
+    1) The traps have to be aligned horizontally.
+    2) Calibrate the force
+    3) Set the scan options to a continuous kymograph and configure the kymograph,
+    ie make sure the correct location is being scanned when making a kymograph
+    4) Make sure the traps are not too close together for catching beads
+
+Workflow of the script
+    1) Catch beads
+    2) Catch a DNA tether
+    3) Check if it is only a single tether, if not, try to break tethers until there is 1 left
+    4) Move the microstage to 'J1' 
+    5) Move the microstage to the protein channel 
+    6) Set the force to 10 pN
+    7) Start a kymograph
+    8) Stop the kymograph and export the data
+    9) Restart step 1 to 8 n times, where n is the number of kymographs you want to make.
+"""
+import os
+import time
+from datetime import datetime
+from pathlib import Path
+
+import bluelake as bl
+import numpy as np
+
+name_bead_channel = "beads"  # This has to match the correct channel name in the UI
+name_dna_channel = "DNA"  # This has to match the correct channel name in the UI
+name_protein_channel = "Ch4"
+name_junction = "J1"
+name_buffer_channel = "buffer"
+time_to_wait_for_flow = 5.0 # [s]
+pressure_flow = 0.25  # Pressure when catching beads and fishing for DNA [bar]
+dna_length_um = 0.34*37.8  # Length of DNA in micrometers [um]
+kymo_duration = 300  # [s]
+
+distance = bl.timeline["Distance"]["Distance 1"]
+force = bl.timeline["Force LF"]["Trap 2"]
+match_score1 = bl.timeline["Tracking Match Score"]["Bead 1"]
+match_score2 = bl.timeline["Tracking Match Score"]["Bead 2"]
+
+
+def throw_if_beads_lost(match_threshold):
+    """Raise an exception if we lose the beads."""
+    if match_score1.latest_value < match_threshold or match_score2.latest_value < match_threshold:
+        raise RuntimeError("Lost beads")
+
+def start_flow(pressure):
+    print("Starting Flow.")
+    bl.fluidics.set_pressure(pressure)
+    bl.fluidics.open(1, 2, 3, 6)
+
+
+def stop_flow():
+    print("Stopping flow.")
+    bl.fluidics.close(1, 2, 3, 6)
+
+
+def catch_beads(match_threshold, pressure=pressure_flow):

```

Comment:
What made you decide to make `pressure` a parameter, but not `time_to_wait_for_flow` for example?